README.md

```diff
@@ -26,19 +26,32 @@ The ```winetricks``` package should be used if it is available and up to date. T
 * Slackbuilds (Slackware): http://slackbuilds.org/repository/14.2/system/winetricks/?search=winetricks
 * Ubuntu: https://packages.ubuntu.com/search?keywords=winetricks
 
-Note: packaged Debian / Ubuntu winetricks versions are typically outdated, so a manual installation is recommended.
-
 If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done.
+
+# Quick install
+
+To quickly install winetricks open a terminal and type
+~~~~~
+wget  https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
+~~~~~
+
+You can then run it from a terminal with
+~~~~~
+sh winetricks
```

Comment:
I will not advise it like this. I would first use chmod and just execute script as intended..

```suggestion
# Make executable
chmod +x ./winetricks

# Run
./winetricks
```

Reply:
See new comment below. But thanks @melroy89